.github/renovate.json

```
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["group:allNonMajor", "schedule:monthly"],
+  "packageRules": [
+    {
+      "description": "Create a PR whenever there is a new major version",
+      "matchUpdateTypes": [
+        "major"
+      ]
+    }
+  ],
+  "ignorePaths": [
+    "examples/**"
+  ]
+}
```

```suggestion
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base",
    ":disableDependencyDashboard"
  ],
  "ignorePaths": [
    ".kokoro/**"
  ],
  "schedule": ["before 8am"],
  "timezone": "Europe/London"
}
```

That's close to what we're using in https://github.com/googleapis/google-cloudevents-dotnet - but in .NET we've been tending to only update dependencies where we really need to. The suggested configuration is likely to suggest more changes than we actually want, so will need tweaking over time, but is a good starting point.